index.html

```diff
@@ -192,7 +192,7 @@
                         <h3>Partnerships</h3>
                         <dl>
                             <dt><a href="http://phet.colorado.edu//">PhET</a></dt>
-                            <dd>Floe is working with PhET to improve the accessibility of their interactive simulations, <a href="http://wiki.fluidproject.org/display/fluid/PhET+Forces+and+Motion+Simulation+Design">improving designs for accessibility</a>.</dd>
+                            <dd>Floe is working with PhET to improve the accessibility of their interactive simulations, <a href="https://wiki.fluidproject.org/x/oQKBAg">improving designs for accessibility</a>.</dd>
```

Comment:
The URL should be replaced with the Sims landing page on the wiki:
https://wiki.fluidproject.org/display/fluid/Inclusivity+and+Accessibility+of+Interactive+Web+Games+and+Simulations